AzureCalling/Views/ParticipantView.swift

```diff
@@ -20,9 +20,11 @@ class ParticipantView: UIView {
     // MARK: IBOutlets
 
     @IBOutlet var contentView: UIView!
+    @IBOutlet weak var containerView: UIView!
     @IBOutlet weak var placeholderImage: UIImageView!
     @IBOutlet weak var videoViewContainer: UIView!
     @IBOutlet weak var activeSpeakerView: ActiveSpeakerView!
+    @IBOutlet weak var switchCameraButton: UIRoundedButton!
```

Comment:
Should these newly added properties only belong to LocalParticipantView?

Reply:
They're referencing outlets in the ParticipantView.xib storyboard. Can't seem to connect those properties if we move them to LocalParticipantView, unless we duplicate the storyboard into LocalParticipantView.xib and remove any camera switch elements in ParticipantView.xib.

@AmyL219 you have any idea how to connect the outlets outside of the File Owner's class without duplicating the storyboard? Or that's the only option to have a clear separation of concerns.

Reply:
Programmatically configure the button in LocalParticipantView instead of relying on ParticipantView.xib with the latest commit. That should remove any references to camera switch in ParticipantView